### ISSUE TITLE
Crazy Bug, Allows you to skip EVERY SINGLE QUEST.

### ISSUE THREAD

**Issue by Awesomedudeperiodunderscore** [Dddatt/bss#23]
I dont know why, but whenever you click on a cannon/slingshots gui instead of pressing e, it brings you to the next line of dialogue on the thing you last interacted with. Pair this with an overnight macro, and well, crazy stuff happens. I currently have six septendecillion honey after macroing brown bears quest. However, this is also starting to make my save file unplayable. You see, most mobs have stopped spawning. This makes doing my only quest left (Riley Bee) impossible to do as i need to kill scorpions that stopped spawning.

Heres a link containing my save file:

[](https://www.khanacademy.org/computer-programming/6-septendecillion-honey/6054698997694464)

**Comment by Awesomedudeperiodunderscore** [comment]
looks like the link didnt go through, so here https://www.khanacademy.org/computer-programming/6-septendecillion-honey/6054698997694464

**Comment by Dddatt** [comment]
Yeah, it's a known bug that I've already fixed in the upcoming update. It's the touch function of the button not being updated properly and becomes out of sync with the E function(which is almost always used, making the glitch uncommon). I don't feel bothered to do anything about it right now cuz I'm too lazy \*-\*

**Comment by spritestatistics** [comment]
crazy
